app/forms/finance/ecf/change_training_status_form.rb

```diff
@@ -77,8 +79,9 @@ def participant_class_name
       end
 
       def action_class_name
```

Comment:
Could you remove this and: `"Participants::#{action_class_name}::#{participant_class_name}".constantize` as this is no longer needed as all the actions have now moved to the refactored services, same in other form

Reply:
Hi @cwrw addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2639/commits/7b435cea9eb74660da58554454c517aff2e94077 tksss 👍 😄